controllers/kuadrant_controller.go

```diff
@@ -135,11 +135,7 @@ func (r *KuadrantReconciler) Reconcile(eventCtx context.Context, req ctrl.Reques
 		logger.V(1).Error(gwErr, "Reconciling cluster gateways failed")
 	}
 
-	specResult, specErr := r.reconcileSpec(ctx, kObj)
-	if specErr == nil && specResult.Requeue {
```

Comment:
what if `reconcileSpec` returns requeue? It never does return such thing?

Reply:
I see it does not. We can add it later if needed. I would not remove this, but not against it either. :+1:

Reply:
I'm in favour of removing this as the golang-ci linter detected is as an linter issue since `reconcileSpec` never returns requeue